source/partials/_nav.slim

```diff
@@ -5,20 +5,21 @@
   = nav_overlay_item('Apprenticeship', '/apprenticeship')
   = nav_overlay_item('Blog', '/blog')
 
-nav#main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}" aria-label=('Main navigation')
-  = link_to '/', class: 'Nav-logo'
-    .NavLogo class=nav_logo_class
+header
+  #main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}"
+    = link_to '/', "aria-label" => "Homepage", class: 'Nav-logo'
+      .NavLogo class=nav_logo_class
 
-  .Nav-menu
-    .Burger class=nav_burger_class
-      span.Burger-line
-      span.Burger-line
-      span.Burger-line
+    .Nav-menu
+      .Burger class=nav_burger_class
+        span.Burger-line
+        span.Burger-line
+        span.Burger-line
 
-  ul.Nav-items
-    = nav_item('Work', '/work')
-    = nav_item('Company', '/company')
-    = nav_item('Community', '/community')
-    = nav_item('Apprenticeship', '/apprenticeship')
-    = nav_item('Blog', '/blog')
-.u-navPlaceholder
+    nav aria-label='Main navigation' role='navigation'
```

Comment:
It seems that having an `ul` inside of the `nav` is definitely the way to go. 

> Note: In cases where the content of a nav element represents a list of items, use list markup to aid understanding and navigation.

http://www.w3.org/TR/html5/sections.html#the-nav-element